src/table/selections-utils.js

```diff
@@ -0,0 +1,79 @@
+export function addSelectionListeners(api, selDispatch) {
```

Comment:
This file isn't all new, just renamed. `addSelectionListeners` and `reducer` are basically what is added